Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs

```diff
@@ -1070,7 +1070,7 @@ public override void Initialize()
             if (ZeroError(CylinderStrokeM, "CylinderStroke"))
                 CylinderStrokeM = 1;
             if (ZeroError(DriverWheelRadiusM, "WheelRadius"))
-                DriverWheelRadiusM = 1;
+                DriverWheelRadiusM = 30.0f;
```

Comment:
What's the reasoning behind the number 30?

Reply:
An arbitrary value.  Wheel radius of loco drive wheels can be anywhere from about 10" to 40". That is why it is important for users to insert the correct value for the locomotive.

Reply:
I see. I suggest dropping a comment that reads `// Wheel radius of loco drive wheels can be anywhere from about 10" to 40".`. That will make this default more understandable.

Reply:
Done